README.md

```diff
@@ -30,10 +30,49 @@ Or **[try it online](https://stackblitz.com/edit/rxjs-autorun-repl?file=index.ts
 
 - `autorun` internally subscribes to `computed` and returns the subscription
 
+### 👓 Tracking
+
+Your can access values from Observables inside `computed` (or `autorun`) in two ways:
+
 - `$(O)` tells `computed` that it should be re-evaluated when `O` emits, with it's latest value
 
 - `_(O)` still provides latest value to `computed`, but doesn't enforce re-evaluation with `O` emission
 
+### 💪 Strength
+
+Some times you want to manage what to do with **subscription of an Observable that is not currently used**.
+
+So we provide three levels of subscription strength:
+
+- `normal` - default - will unsubscribe if the latest run of expression didn't use this observable
+
+  ```ts
+  compute(() => $(a) ? $(b) : 0)
+  ```
+
```

Comment:
Maybe add that you can be explicit about normal binding too?

Which is equivalent to

```ts
compute(() => $.normal(a) ? $.normal(b) : 0)
```